sections/shared/Layout/SideNav/SideNav.tsx

```diff
@@ -51,22 +60,46 @@ const SideNav: FC = () => {
 	const menuLinks = totalBalancePendingMigration > 0 ? MIGRATE_MENU_LINKS : MENU_LINKS;
 
 	return (
-		<SideNavContainer>
+		<SideNavContainer
+			onMouseLeave={() => setSubMenuConfiguration({ ...subMenuConfiguration, routes: null })}
+		>
 			<StakingLogoWrap>
 				<Link href={ROUTES.Home}>
-					<a>
-						<Svg src={StakingLogo} />
-					</a>
+					<Svg src={StakingLogo} />
 				</Link>
 			</StakingLogoWrap>
 			<MenuLinks>
-				{menuLinks.map(({ i18nLabel, link }) => (
+				{menuLinks.map(({ i18nLabel, link, subMenu }, i) => (
 					<MenuLinkItem
+						ref={(r) => {
+							if (subMenu) {
+								menuLinkItemRefs.current = { ...menuLinkItemRefs.current, [i]: r };
+							}
+						}}
+						onMouseEnter={() => {
+							setSubMenuConfiguration(
+								subMenu
+									? {
+											routes: subMenu as any,
```

Comment:
When a MenuItem is hovered we get its `boundingClientRect().y` and save it to the `subMenuConfiguration` object